src/test/java/com/commercetools/sync/types/utils/typeactionutils/BuildLocalizedEnumUpdateActionsTest.java

```diff
@@ -0,0 +1,205 @@
+    private static final List<LocalizedEnumValue> ENUM_VALUES_ABD = asList(ENUM_VALUE_A, ENUM_VALUE_B, ENUM_VALUE_D);
+    private static final List<LocalizedEnumValue> ENUM_VALUES_ABCD = asList(
+        ENUM_VALUE_A,
+        ENUM_VALUE_B,
+        ENUM_VALUE_C,
+        ENUM_VALUE_D
+    );
+    private static final List<LocalizedEnumValue> ENUM_VALUES_CAB = asList(ENUM_VALUE_C, ENUM_VALUE_A, ENUM_VALUE_B);
+    private static final List<LocalizedEnumValue> ENUM_VALUES_CB = asList(ENUM_VALUE_C, ENUM_VALUE_B);
+    private static final List<LocalizedEnumValue> ENUM_VALUES_ACBD = asList(
+        ENUM_VALUE_A,
+        ENUM_VALUE_C,
+        ENUM_VALUE_B,
+        ENUM_VALUE_D
+    );
+    private static final List<LocalizedEnumValue> ENUM_VALUES_ADBC = asList(
+        ENUM_VALUE_A,
+        ENUM_VALUE_D,
+        ENUM_VALUE_B,
+        ENUM_VALUE_C
+    );
+    private static final List<LocalizedEnumValue> ENUM_VALUES_CBD = asList(ENUM_VALUE_C, ENUM_VALUE_B, ENUM_VALUE_D);
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithEmptyPlainEnumValuesAndNoOlEnumValues_ShouldNotBuildActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            Collections.emptyList(),
+            Collections.emptyList()
+        );
+
+        assertThat(updateActions).isEmpty();
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithNewPlainEnumValuesAndNoOldPlainEnumValues_ShouldBuild3AddActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            Collections.emptyList(),
+            ENUM_VALUES_ABC
+        );
+
+        assertThat(updateActions).containsExactly(
+            AddLocalizedEnumValue.of(FIELD_NAME_1, ENUM_VALUE_A),
+            AddLocalizedEnumValue.of(FIELD_NAME_1, ENUM_VALUE_B),
+            AddLocalizedEnumValue.of(FIELD_NAME_1, ENUM_VALUE_C)
+        );
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithIdenticalPlainEnum_ShouldNotBuildUpdateActions() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_ABC
+        );
+
+        assertThat(updateActions).isEmpty();
+    }
+
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithOnePlainEnumValue_ShouldBuildAddEnumValueAction() {
+        final List<UpdateAction<Type>> updateActions = buildLocalizedEnumValuesUpdateActions(
+            FIELD_NAME_1,
+            ENUM_VALUES_ABC,
+            ENUM_VALUES_ABCD
+        );
+
+        assertThat(updateActions).containsExactly(
+            AddLocalizedEnumValue.of(FIELD_NAME_1, ENUM_VALUE_D)
+        );
+    }
+
+    @Test
+    public void buildLocalizedEnumUpdateActions_WithOneEnumValueSwitch_ShouldBuildRemoveAndAddEnumValueActions() {
```

Comment:
ShouldBuildRemoveAndAddEnumValueActions -  I see only 1 action. Also check other methods with this remove thing

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/304/commits/f4fc41525cbd101ff25de148431849084815c98a#diff-cf0202967923903ffb0ed23ca81c0c73R102